### ISSUE TITLE
Live blog does not work on AMP cache

### ISSUE THREAD

**Issue by ithinkihaveacat** [ampproject/amp-by-example#468]
When loaded via <https://cdn.ampproject.org/c/s/ampbyexample.com/samples_templates/live_blog/preview/> the live blog does not appear to ever receive updates:

![screen shot 2016-11-05 at 11 51 04](https://cloud.githubusercontent.com/assets/51244/20029805/58533252-a34e-11e6-80d9-0c227f01f122.png)

**Comment by kul3r4** [comment]
This sample doesn't work in the cache because we are using a cookie for emulating new content.

**Comment by cramforce** [comment]
Maybe you could rewrite the server to always provide a constant number of responses (say 10), but add new entries every 30-60 seconds?

**Comment by kul3r4** [comment]
This is possible, but I would never know where the user starts. The cookie strategy was allowing me to start always from 1. If we could use the CLIENT_ID then we could realize the same mechanism.

**Comment by cramforce** [comment]
Yeah, I understand. You can't get the CLIENT_ID as the point of `amp-live-list` is that it is not user specific.

It may not be possible to build the perfect demo for this.

**Comment by jkingyens** [comment]
amp-live-list might have issues with the cache (http://stackoverflow.com/questions/40411602/how-to-control-amp-live-list-caching). Maybe worth building a sample that works with the cache to find out?

**Comment by dknecht** [comment]
There should be a notice on this page that says it doesn't work on cache.

**Comment by sebastianbenz** [comment]
@dknecht good idea - we should do this.

**Comment by stoneshower** [comment]
Sorry but please let me clarify. In this moment, does  amp-live-list work when the page is loaded from Google AMP Cache? In my case, the "update" button appears only on the original URL page. However, on the page from Google AMP Cache, I needed to refresh the page because `<button update>` was not shown.

**Comment by kul3r4** [comment]
Hi,
no at this moment it does not work on cache.

**Comment by spinmar** [comment]
IN my page, when served by Google amp cache, the button is showed but when user tap on it the new content is showd correctly but the button is not hide. Looking at the code it seems that Google puts the button outside the iframe source and probably the amp-live-list.js is not able to hide it. The same example served by our servers has no problem.

**Comment by themeenanaresh** [comment]
This website Completely indexing on Google with live red icon. I means this site Completely working amp live list.

https://www.jansatta.com/education/sarkari-result-2019-sarkari-naukri-job-2019-sarkariresult-com-live-updates-rrb-ntpc-admit-card-2019-ibps-clerk-recruitment-2019-notification-sarkari-naukri-2019-railway-railway-12th-pass/1186510/lite


Please Help Me. My AMP Version not show update button after Google Cached. 
My website URL  https://www.todaylatestupdate.in/today-current-affairs-in-hindi-ssc-bank-upsc-state-level-exam-important-current-affairs/amp

**Comment by themeenanaresh** [comment]
Please provide details
